### PR TITLE
fix: upgrade and tweak to new aether@0.9.0-rc

### PR DESCRIPTION
**Pull request by obdulia-losantos**

### CODE REVIEW

File: app/gather/assets/css/_survey-list.scss [hunk 1]
```diff
@@ -19,7 +19,7 @@
  */
 
 .surveys-list {
-  @extend .page-content;
+  @extend .page-content; // sass-lint:disable-line placeholder-in-extend
```
Comment (admbtlr) [inline app/gather/assets/css/_survey-list.scss]: Should we disable this rule globally?

Comment (obdulia-losantos) [reply app/gather/assets/css/_survey-list.scss]: Done!

File: app/gather/assets/css/_survey-edit.scss [hunk 2]
```diff
@@ -114,7 +114,7 @@ input.hidden-file {
   }
 }
 
-.FilteredMultiSelect { // sass-lint:disable-line class-name-format
+.FilteredMultiSelect {
```
Comment (admbtlr) [inline app/gather/assets/css/_survey-edit.scss]: Actually I think this rule is good :D - I think we should keep it

Comment (obdulia-losantos) [reply app/gather/assets/css/_survey-edit.scss]: Reintroduce :wink:

**Comment by obdulia-losantos** [comment]
To review the `standard@12` commit I suggest to hide whitespace changes.

Most of the changes were fixing errors like these:
```text
A space is required after '{'.
A space is required before '}'.
```
The rest were to remove deprecated methods like `assert.equal` and `assert.deepEqual`.

image

File: app/gather/api/views.py [hunk 3]
```diff
@@ -142,21 +152,24 @@ def valid_header(name):
...
                                     data=request.body if request.body else None,
                                     headers=headers,
                                     *args,
                                     **kwargs)
-        return HttpResponse(response, status=response.status_code)
+        http_response = HttpResponse(response, status=response.status_code)
+        # copy from the original response headers the exposed ones
+        # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers
+        # https://fetch.spec.whatwg.org/#http-access-control-expose-headers
+        expose_headers = response.headers.get('Access-Control-Expose-Headers', '').split(', ')
```
Comment (admbtlr) [inline app/gather/api/views.py:168]: This looks like a good compromise - which header(s) is it that you need again?

File: app/gather/templates/base.html [hunk 4]
```diff
@@ -26,10 +27,11 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <meta name="robots" content="NONE,NOARCHIVE" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="">
-    <meta name="author" content="">
-    <link href="https://fonts.googleapis.com/css?family=Asap:400,400i|Quicksand:400,500,700" rel="stylesheet">
+    <meta name="description" content="{% trans 'Survey collection and analytics' %}">
```
Comment (admbtlr) [inline app/gather/templates/base.html]: 'Effortless data collection and curation'

**Review by admbtlr — CHANGES_REQUESTED** [review]

File: app/gather/assets/apps/utils/avro-utils.spec.jsx [hunk 5]
```diff
@@ -102,12 +102,12 @@ describe('AVRO utils', () => {
         ]
       }
       const expected = {
-        labels: {first: 'The first'},
+        labels: { first: 'The first' },
         paths: ['first']
       }
-      assert.deepEqual(extractPathDocs(schema), expected)
+      assert.deepStrictEqual(extractPathDocs(schema), expected)
```
Comment (admbtlr) [inline app/gather/assets/apps/utils/avro-utils.spec.jsx:108]: Any specific reason for making all these strict, or are you just being extra careful?

Comment (obdulia-losantos) [reply app/gather/assets/apps/utils/avro-utils.spec.jsx:108]: Nop, `equal`/`deepEqual` were deprecated long time ago and node suggests or enforces to use the `strict` method.

https://nodejs.org/api/assert.html#assert_assert_equal_actual_expected_message

File: app/gather/api/views.py (hunk 3 above)
Comment (obdulia-losantos) [reply app/gather/api/views.py:168]: `Content-Disposition`

https://github.com/eHealthAfrica/aether/pull/336/files#diff-d2d8c6bbb578ea8da3c46269684afe0eR122

File: app/gather/templates/base.html (hunk 4 above)
Comment (obdulia-losantos) [reply app/gather/templates/base.html]: In `README` too?

File: app/gather/assets/apps/utils/avro-utils.spec.jsx (hunk 5 above)
Comment (admbtlr) [reply app/gather/assets/apps/utils/avro-utils.spec.jsx:108]: Ah, should've read your original comment ...

**Review by admbtlr — APPROVED** [review]

**Pull request merged**